internal/util/vault.go

```diff
@@ -244,6 +285,9 @@ Vault will verify service account jwt token with Kubernetes and return token
 if the requester is allowed.
 */
 func (kms *VaultKMS) getAccessToken() (string, error) {
+	if kms.VaultToken != "" {
```

Comment:
Should this function not set `kms.VaultToken` too? If it gets set elsewhere, it should probably get explained in a commit.